src/mechaclient.py

```diff
@@ -126,7 +135,7 @@ def fact_manager(self) -> FactManager:
         This is initialized in a lazy way to increase overall startup speed.
         """
         if not self._fact_manager:
-            self._fact_manager = FactManager()  # Instantiate Global Fact Manager
+            self._fact_manager = FactManager(fact_table="converted_facts")  # Instantiate Global Fact Manager
```

Comment:
[Error] this as discussed is kind of hacky.  The fact manager wasn't designed to be given a table for security reasons, but rather than hard code the name it needs to be moved to a config option.

Reply:
I also agreed to take care of this, so this should not be considered blocking.

Reply:
SPARK-167